src/cpp/src/image_generation/flux_fill_pipeline.hpp

```diff
@@ -0,0 +1,267 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <cassert>
+
+#include "image_generation/diffusion_pipeline.hpp"
+#include "image_generation/flux_pipeline.hpp"
+
+namespace ov {
+namespace genai {
+
+class FluxFillPipeline : public FluxPipeline {
```

Comment:
we have a code to create Image2Image and Text2Image from Inpainting pipelines:
https://github.com/openvinotoolkit/openvino.genai/blob/5636312c4c11178223381bdf408ca0c938b1cf33/src/cpp/src/image_generation/image2image_pipeline.cpp#L56-L70

It asserts here:
https://github.com/openvinotoolkit/openvino.genai/blob/5636312c4c11178223381bdf408ca0c938b1cf33/src/cpp/src/image_generation/flux_pipeline.hpp#L228-L235

based on `is_inpainting_model()`

A question - what will be returned by `is_inpainting_model()` in case of FLUX fill? 
As I see from FLUX fill configs - it will not be detected as inpainting, should we override `is_inpainting_model` explicitly then?